app/models/event.rb

```diff
@@ -0,0 +1,13 @@
+class Event < ApplicationRecord
+
+  belongs_to :center
+
+ # validates :center_id, presence: true
```

Comment:
it's better to validate the object itself. `validates :center, presence: true`. Reference: http://railsguides.net/belongs-to-and-presence-validation-rule1/

@G115H and @lingadingdong take note of this too